discovery/consul/consul.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 
 	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/util/httputil"
+	"net/http"
```

Comment:
Gofmt (which is calledby our make target) doesn't move package imports between import blocks. Calling make/gofmt won't help here.

@YKlausz please move that import up to the other imports of stdlib and keep the list ordered (which gofmt will enforce).